fe/src/project/AspNetCore/shared/components/Files/Files.scss

```diff
@@ -0,0 +1,10 @@
+.file-tooltip {
```

Comment:
**suggestion:** try to stick to BEM convention: http://getbem.com/naming/ It should be .files__tooltip

Reply:
Thanks